script/review-logs.ts

```diff
@@ -0,0 +1,17 @@
+/* tslint:disable:no-sync-functions */
+
+import * as fs from 'fs'
+import * as path from 'path'
+const { getUserDataPath } = require('./dist-info')
```

Comment:
Yeah, I'm sure this could easily turn into a cat bathtub situation.

![giphy](https://user-images.githubusercontent.com/13760/28689980-8f5065b0-72e5-11e7-8978-3e621ba69739.gif)